GPU/GPUTracking/TPCClusterFinder/GPUTPCClusterFinder.h

```diff
@@ -20,6 +20,10 @@
 #include "GPUDataTypes.h"
 #include "CfFragment.h"
 
+#ifdef GPUCA_HAS_ONNX
+#include "GPUTPCNNClusterizerInternals.h"
```

Comment:
please forward-declare the class here, don't include it.
And you don't need to #ifdef-protect it here.